docs/BasicCommunication/OnAppCapabilityUpdated/idex.md

```diff
@@ -0,0 +1,119 @@
+## OnAppCapabilityUpdated
+
+Type
+: Notification
+
+Sender
+: SDL
+
+Purpose
+: Inform that that a specific application capability has changed
+
+### Notification
+
+SDL sends OnAppCapabilityUpdated notification to HMI to provide supported by application <abbr title="Video Streaming Capabilities">VSC</abbr>
+
+!!! may
+
+HMI may switch video streaming resolution according to values provided by application in OnAppCapabilityUpdated notification. (_see also [OnSystemCapabilityUpdated](../OnSystemCapabilityUpdated/index.)_)
```

Comment:
If HMI doesn't receive OnAppCapability updated - HMI should not send OnSysCapabilityUpdated and the user should not be able to change the resolution

Reply:
of if HMI received OnAppCapabilityUpdated without additionalVideoStCaps inside